sass/_mixins.scss

```diff
@@ -276,6 +277,30 @@ $modifiers: (
   speak: none;
 }
 
+/**
+ * Show icons only, hide copy.
+ */
+@mixin icon-only {
```

Comment:
The `@mixin icon-only` hides the copy from any element where you want to display only the icon. This is an accessibility friendly solution alternative for the .sr-only class without the need of having extra markup.